plugins/serializers/ubazzar/ubazzar_json.go

```diff
@@ -0,0 +1,135 @@
+package ubazzar
```

Comment:
```suggestion
package ubazaar
```

Need to rename the directory and file too